rootdir/root/init.qcom.rc

```diff
@@ -460,7 +424,6 @@ service qmuxd /system/bin/qmuxd
 
 service netmgrd /system/bin/netmgrd
     class main
-    disabled
```

Comment:
Ok. Just noticed that init.environ.rc, which set PATH variable amongst other things, is blamed for making android container crashes and is commented out of init.rc. This makes many of shell command (for example, start command which starts a service in the Android container) not working at all. I'll find out why init.environ.rc crashes the container and re-adding it to init.rc.